application-diagram-ui/src/main/resources/Diagram/DiagramEditSheet.xml

```diff
@@ -1171,6 +1171,46 @@ define('diagram-editor', [
     }
   };
 
+
+  /*
+   * Map with all the menu items that we want to have a title.
+   */
+
+  const titleMap = new Map([
+      ['print', 'When you click this button, a popup will appear, allowing you to select which part of the diagram you want to print or export as a PDF.\
+        After clicking the "Print button, a new window will open, displaying a preview of the document. In this window, you can choose to either print the document or save it as a PDF']
```

Comment:
This is too long for a title, but rather something that could be added in documentation, in order to explain some steps and give guidance. Here you just need an explanation. I suggest the following:
`Print the diagram using an external device when possible, otherwise simply export and save it as PDF`

Also, this should be part of a translation